src/lib/openapi/spec/tag-with-version-schema.ts

```diff
@@ -6,9 +6,13 @@ export const tagWithVersionSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['version', 'tag'],
+    description:
+        'A tag with a version number representing the schema used to model the tag.',
     properties: {
         version: {
             type: 'integer',
+            description: 'The version of the schema used to model the tag.',
+            example: 1,
```

Comment:
Same as above. Can we and do we want to use enums here?

Reply:
I don't think it's a good idea to model auto-incremental values with enums